### PR TITLE
#1986 [bet.in] Remove Beteiligung.in/ in header styling

### PR DESCRIPTION
**Pull request by phillimorland**

### CODE REVIEW

File: liqd_product/templates/a4dashboard/base_dashboard.html [hunk 1]
```diff
@@ -8,7 +8,7 @@
     <div class="l-wrapper">
         <div class="platform-brand main-header__brand">
             {% spaceless %}
-                <a href="{% if view.organisation.partner %}{% url 'partner' partner_slug=view.organisation.partner.slug %}{% else %}/{% endif %}" class="platform-brand__link">{% if view.organisation.partner %}<span class="marked">{{ view.organisation.partner.name }}</span>{% else %}Beteiligung.in{% endif %}</a>
+                <a href="{% if view.organisation.partner %}{% url 'partner' partner_slug=view.organisation.partner.slug %}{% else %}/{% endif %}" class="platform-brand__link">{% if view.organisation.partner %}<h2>{{ view.organisation.partner.name }}</h2>{% else %}Beteiligung.in{% endif %}</a>
```
Comment (fuzzylogic2000) [inline liqd_product/templates/a4dashboard/base_dashboard.html]: this should also get the heading--sm and look the same as the partner header (apart from colours and other links)

File: liqd_product/assets/scss/components/_headings.scss [hunk 2]
```diff
@@ -3,3 +3,7 @@
     text-transform: uppercase;
     letter-spacing: 0.1em;
 }
+
+.heading--sm {
```
Comment (fuzzylogic2000) [inline liqd_product/assets/scss/components/_headings.scss]: hmm, this seems to be very specific?! And is not a variant. Ehm, so it either needs another name or might be better to move it to the main_header-component. main-header__brand-something?

Comment (phillimorland) [reply liqd_product/assets/scss/components/_headings.scss]: is small not a modifier?

File: liqd_product/templates/a4dashboard/base_dashboard.html (hunk 1 above)
Comment (phillimorland) [reply liqd_product/templates/a4dashboard/base_dashboard.html]: ahh yes, thanks!

File: liqd_product/assets/scss/components/_headings.scss (hunk 2 above)
Comment (fuzzylogic2000) [reply liqd_product/assets/scss/components/_headings.scss]: yes, but to use a modifier you have to use both. the class and its modifier

File: liqd_product/assets/scss/components/_headings.scss [hunk 3]
```diff
@@ -3,3 +3,7 @@
     text-transform: uppercase;
     letter-spacing: 0.1em;
 }
+
+.heading--sm-margin {
```
Comment (fuzzylogic2000) [inline liqd_product/assets/scss/components/_headings.scss]: Sorry, still no modifier! I would have called it heading-sm-margin. But I will merge anyway!

**Pull request merged**